### PR TITLE
Update renamed json files to new endpoints

### PR DESCRIPTION
**Pull request by ywkim312**
Please test the following endpoints.
Note that the summary endpoint in SNAP is just a place holder using summary.json from SNAP data-import repo, but the content of the data is not the correct summary.

Title I
Subtitle A and others (ARC, PLC)
http://localhost:5000/pdl/titles/title-i/subtitles/subtitle-a/map
http://localhost:5000/pdl/titles/title-i/subtitles/subtitle-a/state-distribution
http://localhost:5000/pdl/titles/title-i/subtitles/subtitle-a/summary

Subtitle D
http://localhost:5000/pdl/titles/title-i/subtitles/subtitle-d/state-distribution
http://localhost:5000/pdl/titles/title-i/subtitles/subtitle-d/summary

Subtitle E
http://localhost:5000/pdl/titles/title-i/subtitles/subtitle-e/state-distribution
http://localhost:5000/pdl/titles/title-i/subtitles/subtitle-e/summary

Title II
EQIP
http://localhost:5000/pdl/titles/title-ii/programs/eqip/map
http://localhost:5000/pdl/titles/title-ii/programs/eqip/state-distribution
http://localhost:5000/pdl/titles/title-ii/programs/eqip/summary

CSP
http://localhost:5000/pdl/titles/title-ii/programs/csp/map
http://localhost:5000/pdl/titles/title-ii/programs/csp/state-distribution
http://localhost:5000/pdl/titles/title-ii/programs/csp/summary

CRP
http://localhost:5000/pdl/titles/title-ii/programs/crp/state-distribution
http://localhost:5000/pdl/titles/title-ii/programs/crp/summary

ACEP
http://localhost:5000/pdl/titles/title-ii/programs/acep/state-distribution
http://localhost:5000/pdl/titles/title-ii/programs/acep/summary

RCPP
http://localhost:5000/pdl/titles/title-ii/programs/rcpp/state-distribution
http://localhost:5000/pdl/titles/title-ii/programs/rcpp/summary

TITLE IV
SNAP
http://localhost:5000/pdl/titles/title-iv/programs/snap/state-distribution
http://localhost:5000/pdl/titles/title-iv/programs/snap/summary

TITLE XI
SNAP
http://localhost:5000/pdl/titles/title-xi/programs/crop-insurance/state-distribution
http://localhost:5000/pdl/titles/title-xi/programs/crop-insurance/summary

### CODE REVIEW

File: app/controllers/pdl.py [hunk 1]
```diff
@@ -548,7 +557,7 @@ def titles_title_i_subtitles_subtitle_a_state_distribution_search():
 
 
 # /pdl/titles/title-i/subtitles/subtitle-a/subprograms:
```
Comment (sandeep-ps) [inline app/controllers/pdl.py:559]: Would you prefer to update these comments, too?

Comment (ywkim312) [reply app/controllers/pdl.py:559]: What do you mean?

Comment (ywkim312) [reply app/controllers/pdl.py:559]: Updated

**Comment by sandeep-ps** [comment]
YongWook and I had a short conversation. He clarified that /pdl/titles/title-iv/programs/snap/summary is a placeholder and we agreed that it probably cannot be used as the data summarizes all titles, not just SNAP. I had missed seeing this detail in the PR description. It's unclear if SNAP needs a summary endpoint, as it doesn't have other subprograms under it. I will approve this PR and have noted this in my Postman collection. Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**